### PR TITLE
Made the compute credit cost tooltip less confusing

### PR DESCRIPTION
**Pull request by RDIL**
Tell the user exactly when compute credit cost will be calculated.

### CODE REVIEW

**Comment by fkorotkov** [comment]
LGTM bu `Prettier` is failing

File: src/components/chips/TaskTransactionChip.tsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ function TaskTransactionChip(props) {
   let { task } = props;
   let { transaction, usedComputeCredits } = task;
   if (!usedComputeCredits) return <div />;
-  let tip = "Exact amount of compute credits used hasn't been calculated yet";
+  let tip = "Compute credit cost will be calculated at the end of the build.";
```
Comment (fkorotkov) [inline src/components/chips/TaskTransactionChip.tsx]: build -> task

```suggestion
  let tip = "Compute credit cost will be calculated at the end of the task.";
```

**Comment by RDIL** [comment]
Prettier failing must be unrelated because it is just string editing 🤔

**Comment by fkorotkov** [comment]
It indicates a failure in this file: https://cirrus-ci.com/task/6606443653890048?command=prettier_check#L6

**Comment by RDIL** [comment]
:/ you beat me

I was just `npm i`ing so I could run it

**Pull request merged**